src/app.ts

```diff
@@ -4,6 +4,8 @@ import { createConnection } from "typeorm";
 import path from "path";
 
 import * as home from "./controllers/home";
+import * as winkels from "./controllers/winkels";
```

Comment:
Ik ben niet zo'n fan van de controllers een Nederlandse naam te geven omdat die in mijn hoofd gelinkt zijn aan het model en niet aan de route.